classes/Gems/Acl/Privilege.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Gems\Acl;
+
+use Gems\Exception\AuthenticationException;
+
+class Privilege
+{
+    public const SEPARATOR = '.';
+
+    public function __construct(
+        public readonly string $name,
+        public readonly ?string $label = null,
+        public readonly ?array $methods = null,
+    )
+    {}
+
+    public function getName(?string $method = null): string
+    {
+        if ($method !== null && $this->methods !== null) {
```

Comment:
Het lijkt me wat raar dat zodra `$this->methods` niet gevuld is, `$method` een dode parameter is?

Reply:
Soms kan een method meegegeven worden, maar de Privilege definitie niet specifiek op method uitsplitsen ($this->methods = null). Dan boeit op dat moment de gegeven method niet.